docs/setup.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "databricks"
+page_title: "Provider: Databricks"
+sidebar_current: "docs-databricks-setup"
+description: |-
+  Databricks sync databricks.
+---
+
+# Databricks-Sync
+
+Databricks-Sync is for workspace disaster recovery. You can use it to import or export some, or all, objects within a workspace. Databricks-Sync can also be used as a migration utility.
+
+## Dependencies
+
+This project requires the following environment dependencies:
+
+* [golang 1.13.x](https://golang.org/dl/)
+* [terraform 0.12.x](https://www.terraform.io/downloads.html)
+* [terraform-provider-databricks 0.2.x](https://registry.terraform.io/providers/databrickslabs/databricks/latest)
+
+## Installation
+
+0. Download: `$ git clone https://github.com/databrickslabs/terraform-provider-databricks.git`
+1. Install Databricks Terraform provider: `$ curl https://raw.githubusercontent.com/databrickslabs/databricks-terraform/master/godownloader-databricks-provider.sh | bash -s -- -b $HOME/.terraform.d/plugins`
```

Comment:
We should move it to another section, this is needed only if they are on terraform 0.12, or do not have internet access

Reply:
I created prerequisites.md in same docs directory